pydatalab/pydatalab/models/people.py

```diff
@@ -90,6 +90,14 @@ def __new__(cls, value):
         return cls.validate(value)
 
 
+class AccountStatus(str, Enum):
+    """A string enum representing the account status."""
+
+    ACTIVE = "active"
+    UNVERIFIED = "unverified"
+    DISABLED = "disabled"
```

Comment:
Just thinking about this again, I think maybe "disabled" was also a bad term, could we replace it with "deactivated" here and in the UI please?